### PR TITLE
Add tests for sizer with default value - omniscidb-internal PR 5403

### PR DESCRIPTION
**Pull request by guilhermeleobas**
As title. Requires PR https://github.com/omnisci/omniscidb-internal/pull/5403

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
dev build is failing because the omniscidb server (5, 7) is equals to the version I'm using in one of the tests. I think it is safe to assume that once omnisci/omniscidb-internal#5403 gets merged and the dev container gets updated, this tests will pass.

**Comment by pearu** [comment]
@guilhermeleobas to handle the out-of-date omniscidb docker failures, you can now (after rebasing) specify the expected date of omniscidb dev version to skip tests using the future features. See the `date` argument in
https://github.com/xnd-project/rbc/blob/e105e1a292e3901f1c4513ade52c5354079c6c19/rbc/tests/test_omnisci_udtf.py#L66
as an example.

**Comment by guilhermeleobas** [comment]
Failures are not related to changes but with an HTTP error in conda

File: rbc/tests/test_omnisci_column_basic.py [hunk 1]
```diff
@@ -509,6 +509,8 @@ def redefined_udtf(x, m, y):  # noqa: E501, F811
             available_version,)))
 @pytest.mark.parametrize("step", [1, 2, 3])
 def test_overload_nonuniform(omnisci, step):
+    pytest.xfail('Test failing due to the introduction of default sizer. See PR 313')
```
[the remark this answers is not in the document]
Comment (pearu) [reply rbc/tests/test_omnisci_column_basic.py:512]: The test passes when using omniscidb master. So, the regression is introduced in https://github.com/omnisci/omniscidb-internal/pull/5403

**Review by pearu — CHANGES_REQUESTED** [review]
Mostly nits. The regression introduced by https://github.com/omnisci/omniscidb-internal/pull/5403 is worrying.

Also, in the future, we should avoid modifying tests that are maintained under omniscidb repository unless the test are made version dependent explicitly. For instance, running the tests of this PR using the master omniscidb will lead to test failures.

**Comment by pearu** [comment]
Merged because it was blocking in testing the master of omniscidb-internal.

The nits need to be resolved in subsequent PRs.

File: rbc/tests/test_omnisci_column_basic.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (guilhermeleobas) [reply rbc/tests/test_omnisci_column_basic.py:512]: > The test passes when using omniscidb master. So, the regression is introduced in omnisci/omniscidb-internal#5403

Not really. The regression was already there, we just didn't have a test covering it. https://github.com/xnd-project/rbc/issues/319 contains a reproducer for this error.

**Pull request merged**